src/Serilog.Sinks.AzureTableStorage/LoggerConfigurationAzureTableStorageExtensions.cs

```diff
@@ -47,7 +48,7 @@ public static class LoggerConfigurationAzureTableStorageExtensions
         /// <param name="loggerConfiguration">The logger configuration.</param>
         /// <param name="storageAccount">The Cloud Storage Account to use to insert the log entries to.</param>
         /// <param name="restrictedToMinimumLevel">The minimum log event level required in order to write an event to the sink.</param>
-        /// <param name="formatProvider">Supplies culture-specific formatting information, or null.</param>
+        /// <param name="formatProvider">Supplies culture-specific formatting information, or null. (Note this is no longer used and a default ITextFormatter of JsonFormatter is used)</param>
```

Comment:
I am not sure this should be ignored- shouldn't we just forward calls from these overloads to the new ones, and pass a `JsonFormatter` as the first argument?